frontend/src/v5/ui/controls/assigneesSelect/assigneesMultiSelect.component.tsx

```diff
@@ -45,21 +45,20 @@ export const AssigneesSelect = ({
 	onBlur,
 	className,
 	...props
-}: IAssigneesSelect) => {
+}: IAssigneesMultiSelect) => {
 	const [open, setOpen] = useState(false);
 
 	// Must filter out users not included in this teamspace. This can occur when a user
 	// has been assigned to a ticket and later on is removed from the teamspace
 	const allUsersAndJobs = UsersHooksSelectors.selectUsersAndJobs();
-	const filteredValues = intersection(value, allUsersAndJobs.map((i) => i.user || i._id));
-
+	// eslint-disable-next-line max-len
+	const filteredValue: string[] = intersection(value, allUsersAndJobs.map((i) => i.user || i._id)) ?? [];
```

Comment:
`filteredValues` has been renamed and lost the `s` at the end, but it is an array, so I think the `s` good to indicate it was an collection